apps/scan/backend/src/scanners/pdi/pdi_app_scan_double_sheet.test.ts

```diff
@@ -344,6 +344,39 @@ test('insert two sheets at once', async () => {
   );
 });
 
+test('insert two sheets at once - scanComplete event', async () => {
+  await withApp(
+    async ({ apiClient, mockScanner, mockUsbDrive, mockAuth, clock }) => {
+      await configureApp(apiClient, mockAuth, mockUsbDrive);
+
+      clock.increment(delays.DELAY_SCANNING_ENABLED_POLLING_INTERVAL);
+      await waitForStatus(apiClient, { state: 'no_paper' });
+
+      mockScanner.emitEvent({ event: 'scanStart' });
+      await expectStatus(apiClient, { state: 'scanning' });
+      // In this case, the scanner seems to set the documentJam flag as well
+      mockScanner.setScannerStatus({
+        ...mockStatus.documentInFront,
+        documentJam: true,
+      });
+      mockScanner.emitEvent({ event: 'error', code: 'doubleFeedDetected' });
+      mockScanner.emitEvent({
+        event: 'scanComplete',
+        images: await ballotImages.completeHmpb(),
```

Comment:
In reality we'd likely be getting really short images, right? Do you want to just create a pair of ~100px tall images for this?

Reply:
I've noticed a variety of behavior actually, from stopping immediately to scanning the entire ballot (granted, that was in a case where I intentionally mis-calibrated double sheet detection by using printed ballots instead of blank sheets)